packages/model/package.json

```diff
@@ -27,6 +27,7 @@
     "ember-cli-string-utils": "^1.1.0",
     "ember-cli-test-info": "^1.0.0",
     "ember-cli-typescript": "^4.1.0",
+    "ember-cache-primitive-polyfill": "^1.0.1",
```

Comment:
Do we need this if implementing ourselves?

Reply:
Yes, it's what polyfills the two helpers that we import for it for 3.20 (they weren't in ember until 3.22)